.github/workflows/publish-packages-main.yaml

```diff
@@ -0,0 +1,106 @@
+name: Publish Package
+
+on:
+  workflow_dispatch:
+    inputs:
+        package:
+          description: 'Name of package to publish'
+          required: true
+          default: 'components'
+          type: choice
+          options:
+          - components
+          - contexts
+          - hooks
+          - resource-utils
+          - utils
+        version:
+          description: 'SemVer format release tag, i.e. 0.2.4'
+          required: true
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
Should this include the workspace? I should be able to run a release for utils and components at the same time?